tests/local.mk

```
@@ -47,7 +65,7 @@ $(kuttl_bin): | $(go_bin)
 	go install github.com/kudobuilder/kuttl/cmd/kubectl-kuttl@latest
 
 test-integration: export KUBECONFIG = $(KIND_KUBECONFIG)
-test-integration: $(kuttl_bin) local-install ## Run integration tests with kuttl
+test-integration: $(kuttl_bin) local-install generate-integration-compositions
```

Only if you add a comment with `##`, it will show up in `make help`
```suggestion
test-integration: $(kuttl_bin) local-install generate-integration-compositions ## Run integration tests with kuttl
```